### ISSUE TITLE
Requesting adding a feature/create a new plugin that let's us anonymize who are watching the replays in game.

### ISSUE THREAD

**Issue by herculuis** [esnar11/SkipReplay#4]
Hi. I saw your skip replay plugin, it's a handy tool. I was wondering if you could make a plugin that let's us not see who are watching the replays after a goal has been scored. In game, many times if we mess up our teammates keep watching the replay to rub it in even more. Now, if we can simply anonymize (let's say with bot names or something else)/remove the names at the bottom of the screen when the replay is shown, we wouldn't know who watched it, maybe the teammate didn't but rather the opponents. We wouldn't know and it would at will reduce tilt and further toxicity. Since we can't force-skip replays cause even if one person is watching it, the whole replay is shown, therefore this could help us out to have a less annoying experience at the very least.
I am requesting simply based on the hopeful assumption that it can be done. I am not a coder, therefore I am not sure if it's even possible to do so, but since you are a developer you probably would know. I am just throwing the idea here.
Thanks.

**Comment by luukvbaal** [comment]
I doubt it this is possible through the bakkesmod API. Even bakkesmod's builtin "Anonymyzer" is reported as "mostly broken".

**Comment by herculuis** [comment]
> I doubt it this is possible through the bakkesmod API. Even bakkesmod's builtin "Anonymyzer" is reported as "mostly broken".

Yeah, however, Anonymizer, although doesn't work completely, still does work with many opponents/teammates. Sometimes it doesn't work at all, I would say when it comes to anonymizing nameplates, it's success rate is about 60-70 percent, but titles and such it can almost never anonymize. I think they gave up on that feature after some updates, maybe too much work or didn't really care for after that. Who knows, maybe with replays it can be done (Hopium)
